lib/report/accounts.rb

```diff
@@ -0,0 +1,30 @@
+module Report
+  class Accounts
+    def self.report(options)
+      new(options).report
+    end
+
+    def initialize(user_id_pepper:)
+      @user_id_pepper = user_id_pepper
+    end
+
+    def report
```

Comment:
🤔 Now we're doing all this I do wonder if we should allow folks the ability to opt out of these reports...
Might ask the PA/UCD crew about this...

Reply:
Done on slack, would be interested to see if there are any takers for that thought before we do the first export?

Reply:
Raise this for a chat next week, not going to make this a blocker here.